wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,108 @@
+# Difficulty Naming
+
+This wiki article serves as a substitute and an update to [this topic](https://osu.ppy.sh/community/forums/topics/178700). 
+
+The purpose of difficulty naming is telling players how difficult a map is. This article lists common conventions as well as other alternatives that have been gathered throughout the years. However, as long as the set of difficulty names is descriptive, you may use any desired kind of scheme. 
+
+## Default	
+
+This subsection shows the default **scaling** difficulty names for each game mode, which are the most commonly used.
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+ 
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Insane](/wiki/shared/diff/insane-s.png "Insane") Insane
+- ![Expert](/wiki/shared/diff/expert-s.png "Expert") Expert
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Kantan](/wiki/shared/diff/easy-t.png "Kantan") Kantan
+- ![Futsuu](/wiki/shared/diff/normal-t.png "Futsuu") Futsuu
+- ![Muzukashii](/wiki/shared/diff/hard-t.png "Muzukashii") Muzukashii
+- ![Oni](/wiki/shared/diff/insane-t.png "Oni") Oni
+- ![Inner Oni / Ura Oni](/wiki/shared/diff/expert-t.png "Inner Oni / Ura Oni") Inner Oni / Ura Oni
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
+
+- ![Cup](/wiki/shared/diff/easy-c.png "Cup") Cup 
+- ![Salad](/wiki/shared/diff/normal-c.png "Salad") Salad
+- ![Platter](/wiki/shared/diff/hard-c.png "Platter") Platter
+- ![Rain](/wiki/shared/diff/insane-c.png "Rain") Rain
+- ![Overdose](/wiki/shared/diff/expert-c.png "Overdose") Overdose
+
+### ![osu!mania](/wiki/shared/Mania.gif "osu!mania") osu!mania
+
+- ![EZ](/wiki/shared/diff/easy-m.png "EZ") EZ 
+- ![NM](/wiki/shared/diff/normal-m.png "NM") NM
+- ![HD](/wiki/shared/diff/hard-m.png "HD") HD
+- ![MX](/wiki/shared/diff/insane-m.png "NX") MX
+- ![SC](/wiki/shared/diff/expert-m.png "SC") SC
+
+## Widely Accepted Alternatives 
+
+The following **individual** difficulty names have been accepted and are used more commonly now as alternative to the basic difficulty names:			
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") **Beginner**: in a default Easy-Normal-Hard-Insane-Expert spread, it means the lowest level even below Easy.
+- ![Advanced](/wiki/shared/diff/normal-s.png "Advanced") **Advanced**: in a default Easy-Normal-Hard-Insane-Expert spread, it's a difficulty between Normal and Hard.
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") **Hyper**: in a default Easy-Normal-Hard-Insane-Expert spread, a difficulty between Hard and Insane.
+- ![Extreme](/wiki/shared/diff/expert-s.png "Extreme") **Extreme**: can be used to represent a difficulty tier above Extra/Expert.
+- ![Marathon](/wiki/shared/diff/expert-s.png "Marathon") **Marathon**: map that is over 5 minutes of [drain time](/wiki/drain_time).
```

Comment:
Marathon difficulties or diffs claiming to have this name usually can stand on their own and form their own spread so this should be default

Reply:
but no one uses Marathon anyways, the fact that this is even mentioned is beyond me but whatever i'll add it to default